CHANGELOG.txt

```diff
@@ -1,3 +1,8 @@
+=== 4.0.0 (unreleased) ===
```

Comment:
I know the PR guidelines suggest to do this but for 4.0.x, let's leave the changelog as is.
Once we're closer to release, then we can use the pr descriptions to compose the changelog.
The reason for this is that otherwise there will be conflicts all the time with changelog file and will need constant rebasing, etc..